airbyte-ci/connectors/pipelines/pipelines/actions/environments.py

```diff
@@ -858,11 +885,16 @@ async def with_airbyte_java_connector(context: ConnectorContext, connector_java_
 
 async def get_cdk_version_from_python_connector(python_connector: Container) -> Optional[str]:
     pip_freeze_stdout = await python_connector.with_entrypoint("pip").with_exec(["freeze"]).stdout()
-    pip_dependencies = [dep.split("==") for dep in pip_freeze_stdout.split("\n")]
-    for package_name, package_version in pip_dependencies:
-        if package_name == "airbyte-cdk":
-            return package_version
-    return None
+    pip_dependencies = [dep for dep in pip_freeze_stdout.split("\n") if "airbyte-cdk" in dep]
+    if not pip_dependencies:
+        return None
+
+    airbyte_cdk_line = pip_dependencies[0]
```

Comment:
Isn't it very brittle to consider the cdk package is always the first in the list?

Reply:
Ah so originally I was filtering the `pip_dependencies` down to only those that contained `airbyte-cdk`

I updated this to make use of `next` so that we dont need to explicitly get an element at index